litho-it/src/test/resources/processor/TestMount.java

```diff
@@ -634,25 +634,29 @@ public Builder prop7(CharSequence prop7) {
     }
 
     public Builder prop7Res(@StringRes int resId) {
-      this.mTestMountImpl.prop7 = resolveStringRes(resId);
+      final CharSequence res = resolveStringRes(resId);
+      this.mTestMountImpl.prop7 = res;
```

Comment:
Would it be possible to collapse the lines here?

Reply:
I did this because `DIMEN_SIZE` `DIMEN_TEXT` and `DIMEN_OFFSET` can take both `Integer` and `Float` but `resolveDimenSize` only return `int`.So when you specific resType as any of them and pass a `Float ` there will be an error. 

I can collapse the lines by adding a cast when there is `Float` type with a `resolveDimenSize` method so that I should add `dimenResBuilders` and `dimenAttrBuilders` to handle this. And Also I will add a test.

Reply:
It's done